packages/ra-ui-materialui/src/list/filter/FilterLiveSearch.spec.tsx

```diff
@@ -30,6 +30,16 @@ describe('FilterLiveSearch', () => {
         fireEvent.click(screen.getByLabelText('ra.action.clear_input_value'));
         expect(screen.queryAllByRole('listitem')).toHaveLength(27);
     });
+    it("input search shouldn't change url on submit", () => {
```

Comment:
Still not good: this test passes even if I comment the preventDefault in `FilterLiveSearch`